recipes/sdsl-lite/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+import yaml
+
+
+class SDSLLite(ConanFile):
+    name = "sdsl-lite"
+    libname = "sdsl"
```

Comment:
Recipe-specific class member should start with an underscore.
It looks like `libname` is used only once, so maybe this member can be removed?